packages/tools/fetch-tool/src/fluidAnalyzeMessages.ts

```diff
@@ -480,63 +480,72 @@ function processOp(
     let recorded = false;
     if (isRuntimeMessage(message)) {
         const runtimeMessage = unpackRuntimeMessage(message);
-        if (runtimeMessage.type === ContainerMessageType.Attach) {
-            const attachMessage = runtimeMessage.contents as IAttachMessage;
-            processDataStoreAttachOp(attachMessage, dataType);
-        } else {
-            let envelop = runtimeMessage.contents as IEnvelope;
-            // TODO: Legacy?
-            if (envelop && typeof envelop === "string") {
-                envelop = JSON.parse(envelop);
+        switch (runtimeMessage.type) {
+            case ContainerMessageType.Attach: {
+                const attachMessage = runtimeMessage.contents as IAttachMessage;
+                processDataStoreAttachOp(attachMessage, dataType);
             }
-            const innerContent = envelop.contents as {
-                content: any;
-                type: string;
-            };
-            const address = envelop.address;
-            type = `${type}/${innerContent.type}`;
-            if (innerContent.type === DataStoreMessageType.Attach) {
-                const attachMessage = innerContent.content as IAttachMessage;
-                let objectType = attachMessage.type;
-                if (objectType.startsWith(objectTypePrefix)) {
-                    objectType = objectType.substring(objectTypePrefix.length);
+            // skip for now because these ops do not have contents
+            case ContainerMessageType.BlobAttach: {}
+            default: {
```

Comment:
I think you missed the break statement. Previous logic had if and else and the else block would not execute if "if" is executed.